client/src/pages/Discussion.stories.jsx

```diff
@@ -23,12 +22,23 @@ const player = {
 
   const round = {
     id: 5,
-    topic: "gun control"
+    data: {"topic": `## Markdown or HTML?
```

Comment:
I wonder if this is where the problem with rendering the discussion stories is. You may need to define the multiline string outside of the data dictionary as `const mdstring = ...`, and then use the new variable in this data dictionary. This may be interpreting the multiline string as a string containing backticks, and the markdown renderer may be printing its contents as code: https://www.markdownguide.org/basic-syntax/#code